varats/utils/cli_util.py

```diff
@@ -95,15 +121,21 @@ def get_research_tool(
         the research tool with the specified ``name``,
         otherwise, raises LookupError
     """
-    if name in ("VaRA", "vara"):
-        return VaRA(
-            source_location if source_location is not None else
-            Path(vara_cfg()["vara"]["llvm_source_dir"].value)
-        )
+    rs_type = get_research_tool_type(name)
 
-    raise LookupError(f"Could not find research tool {name}")
+    if source_location:
+        src_folder = Path(source_location)
+    elif rs_type.has_source_location():
+        src_folder = rs_type.source_location()
+    else:
+        src_folder = Path(os.getcwd()) / "tools_src/"
+
+    if not src_folder.exists():
+        src_folder.mkdir(parents=True)
+
+    return rs_type(src_folder)
 
 
 def get_supported_research_tool_names() -> tp.List[str]:
     """Returns a list of all supported research tools."""
-    return ["VaRA", "vara"]
+    return ["VaRA", "vara", "phasar"]
```

Comment:
I suggest to drop `VaRA` from this list. Instead, you could now specify a `key_normalizer` in the CLI-parser (s. https://github.com/madman-bob/python-argparse-utils). 

```suggestion
    return ["vara", "phasar"]
```